test/Microsoft.TestPlatform.AcceptanceTests/DifferentTestFrameworkSimpleTests.cs

```diff
@@ -20,9 +20,11 @@ public class DifferentTestFrameworkSimpleTests : AcceptanceTestBase
     public void ChutzpahRunAllTestExecution(RunnerInfo runnerInfo)
     {
         SetTestEnvironment(_testEnvironment, runnerInfo);
-        var testJSFileAbsolutePath = Path.Combine(_testEnvironment.TestAssetsPath, "test.js");
-        var arguments = PrepareArguments(testJSFileAbsolutePath, GetTestAdapterPath(UnitTestFramework.Chutzpah), string.Empty, FrameworkArgValue, runnerInfo.InIsolationValue, resultsDirectory: TempDirectory.Path);
-
+        string fileName = "test.js";
```

Comment:
We want to copy the js file into a separate temp dir for each test run so that they don't clash. This solves the issue (only in the logs) about file used by another process:

```
TpTrace Verbose: 0 : 55852, 7, 2022/03/16, 16:19:54.836, 5422429197711, vstest.console.exe, TestRequestSender.OnExecutionMessageReceived: Received message: {"Version":6,"MessageType":"TestSession.Message","Payload":{"MessageLevel":2,"Message":"Error: System.IO.IOException: The process cannot access the file 'C:\\src\\vstest2\\test\\TestAssets\\_Chutzpah.b528605807a2aca41e9f883429df5340b7ada09a.test.html' because it is being used by another process.\r\n at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)\r\n at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)\r\n at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)\r\n at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)\r\n at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)\r\n at System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)\r\n at System.IO.File.WriteAllText(String path, String contents)\r\n at Chutzpah.Wrappers.FileSystemWrapper.Save(String path, String contents) in C:\\dev\\chutzpah\\Chutzpah\\Wrappers\\FileSystemWrapper.cs:line 172\r\n at Chutzpah.TestHarnessBuilder.CreateTestHarness(TestContext testContext, TestOptions options) in C:\\dev\\chutzpah\\Chutzpah\\TestHarnessBuilder.cs:line 78\r\n at Chutzpah.TestRunner.<>c__DisplayClass39_0.<ExecuteTestContexts>b__0(TestContext testContext) in C:\\dev\\chutzpah\\Chutzpah\\TestRunner.cs:line 442\nWhile Running:C:\\src\\vstest2\\test\\TestAssets\\test.js\n"}}
```